### PR TITLE
doc: Added Clone Pipeline Config

### PR DESCRIPTION
**Pull request by ashokdevtron**
- Added **Clone Pipeline Configuration** section in Application Group doc.
- Use Case added
- RBAC added
- Enterprise Tag added
- Snapshots and GIFs added
- Interlinking added in **CD Pipeline** doc in Workflow section

### CODE REVIEW

File: docs/user-guide/application-groups.md [hunk 1]
```diff
@@ -140,6 +140,60 @@ Users need to have [Build and deploy permission](../user-guide/global-configurat
 
 ## Additional Features
 
+### Clone Pipeline Configuration [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Only superadmins can clone pipeline configuration.
```
Comment (abhibhaw) [inline docs/user-guide/application-groups.md:147]: wording needs to be changed if heading gets updated

File: docs/user-guide/application-groups.md [hunk 2]
```diff
@@ -140,6 +140,60 @@ Users need to have [Build and deploy permission](../user-guide/global-configurat
...
+{% endhint %}
+
+This feature aims at helping the user clone existing CI/CD pipelines for new target environments in multiple applications. The configurations present in a given CI/CD pipeline also get copied in the cloned pipelines including the following:
+
+* Scripts/Plugins from Pre-CI, Post-CI, Pre-CD, Post-CD
```
Comment (abhibhaw) [inline docs/user-guide/application-groups.md:152]: we had a list in PRD, what gets cloned and what not, maybe we can add that here

File: docs/user-guide/application-groups.md [hunk 3]
```diff
@@ -140,6 +140,60 @@ Users need to have [Build and deploy permission](../user-guide/global-configurat
...
+**Use Case**: Let's say you had 'n' number of apps deployed to a testing environment named `qa-env1`. Your team size doubled, thus necessitating the addition of another testing environment (`qa-env2`) with those apps deployed. Manually creating pipelines and configuring it for `qa-env2` environment in each app might be impractical. 
+
+#### Methods of Cloning
+
+This feature gives you two methods of cloning:
+1. **New Workflow**: Creates a new workflow and clones the source CI and CD pipeline. Gives you the flexibility to tweak the cloned CI (e.g., changing code branch for build).
+
+    ![Figure 18: New Workflow](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/application-groups/new-workflow.jpg)
+
+2. **Source Workflow**: Uses the same workflow but clones only its existing CD pipeline. Allows you to utilize the original CI source available for source environment.
```
Comment (abhibhaw) [inline docs/user-guide/application-groups.md:165]: the sentence needs more clarification

**Review by abhibhaw — CHANGES_REQUESTED** [review]
We should aim to use(from next time) examples which are relatable to users rather than doc1 and doc2

**Comment by ashokdevtron** [comment]
Copied to the new repo and fixed in this PR: https://github.com/devtron-labs/devtron-documentation/pull/7

**Pull request closed without merging**